src/index.js

```diff
@@ -39,13 +39,17 @@ app.use(passport.session());
 
 const port = process.env.PORT || 3000;
 
-app.use('/api', welcome);
+app.use('/', welcome);
 app.use('/api-doc', swagger);
 app.use('/api/v1/auth', authRouter);
 app.use('/api/v1/users', userRouter);
 app.use('/api/v1/trips', tripsRouter);
 app.use('/api/v1/facilities', facilitiesRouter);
 
+app.use((req, res) => res.status(404).send({ status: 404, error: res.__('Route %s not found', req.url) }));
+
+app.use((err, req, res) => res.status(500).send({ status: 500, error: res.__('SERVER ERROR') }));
```

Comment:
can you change the response message to lowercase
```suggestion
app.use((err, req, res) => res.status(500).send({ status: 500, error: res.__('server error') }));
```

Reply:
I changed it.